client/jetpack-connect/jetpack-connection.jsx

```diff
@@ -272,6 +266,7 @@ const jetpackConnection = ( WrappedComponent ) => {
 					renderFooter={ this.renderFooter }
 					renderNotices={ this.renderNotices }
 					isCurrentUrlFetching={ this.isCurrentUrlFetching() }
+					connectProps={ this.props }
```

Comment:
Perhaps it would be more straightforward to use a spread operator here: `{ ...this.props }`? I can easily see cases where a dev forgets this HoC moves props to `connectProps`. It also would negate the need for the `path` prop above.

Reply:
Resolved in bfdc5f806b465d9bf4792f7ffc4d7a4c89ce2556.